zrml/futarchy/README.md

```diff
@@ -1 +1,46 @@
 # Futarchy Module
+
+The futarchy module provides a straightforward, "no bells and whistles"
+implementation of the [futarchy governance
+system](https://docs.zeitgeist.pm/docs/learn/futarchy).
+
+## Overview
+
+The futarchy module is essentially an oracle based governance system: When a
+proposal is submitted, an oracle is specified which evaluates whether the
+proposal should be executed. The type of the oracle is configured using the
+associated type `Oracle`, which must implement `FutarchyOracle`.
+
+The typical oracle implementation for futarchy is the `DecisionMarketOracle`
+implementation exposed by the neo-swaps module, which allows making decisions
+based on prices in prediction markets. A `DecisionMarketOracle` is defined by
+providing a pool ID and two outcomes, the _positive_ and _negative_ outcome.
+The oracle evaluates positively (meaning that it will allow the proposal to
+pass) if and only if the positive outcome is move valuable than the negative
+outcome.
+
+The standard governance flow is the following:
+
+- A hard-coded origin (usually root) submits a proposal to be approved or
+  rejected via futarchy. If the origin is root, this is done by running a
```

Comment:
The newest version just uses the root origin, not a configurable hard-coded origin anymore.

Reply:
6ec557c94db52b380854bb402d044c0a7eb2ce4f